port/linux/oc_config.h

```diff
@@ -93,6 +93,9 @@ typedef uint64_t oc_clock_time_t;
 /* Maximum number of endpoints */
 #define OC_MAX_NUM_ENDPOINTS (20)
 
+/* Maximum size of uri for a collection resource */
+// #define OC_MAX_COLLECTIONS_INSTANCE_URI_SIZE (64)
+
```

Comment:
please add this macro to all oc_config.h of ports.

Reply:
done

Reply:
hmm, wait this is under

```
#ifndef OC_DYNAMIC_ALLOCATION
```

which is not correct, I'll have to move this.